chart/templates/statsd/statsd-deployment.yaml

```
@@ -87,6 +88,9 @@ spec:
         - name: statsd
           image: {{ template "statsd_image" . }}
           imagePullPolicy: {{ .Values.images.statsd.pullPolicy }}
+          securityContext: {{ $containerSecurityContext | nindent 12 }}
+          args:
+            - "--statsd.mapping-config=/etc/statsd-exporter/mappings.yml"
           {{- if .Values.statsd.args }}
           args: {{ tpl (toYaml .Values.statsd.args) . | nindent 12 }}
           {{- end }}
```

`args` is duplicated
```suggestion
          {{- if .Values.statsd.args }}
          args: {{ tpl (toYaml .Values.statsd.args) . | nindent 12 }}
          {{- else}}
          args:
            - "--statsd.mapping-config=/etc/statsd-exporter/mappings.yml"
          {{- end }}
```

---

thanks for the suggestion!
I added it